.NET/Microsoft.Recognizers.Text.Sequence/Portuguese/PortuguesePhoneNumberExtractorConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions;
+using Microsoft.Recognizers.Definitions.Portuguese;
+
+namespace Microsoft.Recognizers.Text.Sequence.Portuguese
+{
+    public class PortuguesePhoneNumberExtractorConfiguration : PhoneNumberConfiguration
```

Comment:
Can't we have a BasePhoneNumberExtractorConfiguration and each language extends it?
PT only needs to load the new regex differently from base.